.github/workflows/swift-toolchain.yml

```diff
@@ -2416,7 +2410,7 @@ jobs:
                 -D CMAKE_Swift_COMPILER=${SWIFTC} `
                 -D CMAKE_Swift_COMPILER_TARGET=${{ matrix.triple }} `
                 -D CMAKE_Swift_COMPILER_WORKS=YES `
-                -D CMAKE_Swift_FLAGS="${{ inputs.CMAKE_Swift_FLAGS }}" `
```

Comment:
Should this be appended rather than replaced?
(Same for all other instances of this replacement)

Reply:
appended where?

Reply:
Oh nevermind, I was on mobile and misread the diff.